examples/algorithms/BinarySearch/BinarySearch.repl.fsx

```diff
@@ -0,0 +1,18 @@
+﻿#r "../../../bin/ulibfs.dll"
+#r "bin/Debug/BinarySearch.dll"
+open BinarySearch
+
+open FStar_Seq_Base
+
+let sequence = MkSeq ([ 5; 3; 86; 23; 11; 98; 1; ] |> List.map (Prims.of_int))
+let found_98 = bsearch sequence (Prims.of_int 98)
+match found_98 with
+| FStar_Pervasives_Native.Some a -> printfn "Found 98"
+| FStar_Pervasives_Native.None -> failwith "Didn't find 98"
+
+let found_30 = bsearch sequence (Prims.of_int 30)
+match found_30 with
+| FStar_Pervasives_Native.None -> printfn "Didn't find 30"
+| FStar_Pervasives_Native.Some a -> failwith "30 shouldn't exist!"
+
```

Comment:
hi @mateuszbujalski 

Can you explain what these repl.fsx files do?

I suppose they run a repl test in F#. But, would be good to confirm and document their purpose somewhere.

Reply:
It's fairly common to include a fsx file with a project to show how it can be used (or as a playground). 

On naming, there was "Huffman.repl" file which I ported to F# (and appended fsx extension). I must have forgotten to remove the "repl" from the name. I'll rename those files.